### PR TITLE
OpenCL Improvements

### PR DESCRIPTION
**Pull request by benoitsteiner**

### CODE REVIEW

File: tensorflow/core/common_runtime/sycl/sycl_allocator.cc [hunk 1]
```diff
@@ -25,6 +25,9 @@ string SYCLAllocator::Name() { return "device:SYCL"; }
 
 void *SYCLAllocator::AllocateRaw(size_t alignment, size_t num_bytes) {
   assert(device_);
+  if(num_bytes == 0) {
```
Comment (vrv) [inline tensorflow/core/common_runtime/sycl/sycl_allocator.cc]: can you run clang-format over all these files?

**Comment by vrv** [comment]
@pelmers can you sign the CLA with the email you used in your git commit?  Our bot surprisingly didn't flag this.

I believe the others are repeat commiters so we don't need their explicit acknowldgement in every PR (so I've been told).

**Comment by vrv** [comment]
also lots of failures, but I assume you're looking at that.

**Comment by pelmers** [comment]
@vrv submitted my CLA

**Comment by vrv** [comment]
Cool, confirmed, thanks!

**Comment by vrv** [comment]
Same for @krikru

**Comment by lukeiwanski** [comment]
I signed it.

**Comment by vrv** [comment]
@lukeiwanski thanks -- you're a repeat contributor so no need to respond in the future :)

signcla bot is actually broken for this PR for some reason, i just manually inspected the emails of the others and found they weren't signed.

**Comment by lukeiwanski** [comment]
@vrv sorry didn't notice your previous post :)

**Comment by vrv** [comment]
Still need @krikru to sign the CLA under the git commit email he used and then we can merge.

**Comment by krikru** [comment]
Signed!

**Comment by vrv** [comment]
Did you sign with "krikr808@student.liu.se" ? that's what you used in your git commit...

**Comment by yaroslavvb** [comment]
PS: I've used this script with success to change email when my git commits had wrong email address -- https://help.github.com/articles/changing-author-info/

**Comment by krikru** [comment]
@vrv Ah, no, I signed with another email address.

I created a new pull request from my fork of tensorflow-opencl into tensorflow-opencl in which I have changed the email of my commits to the one I signed the CLA with. Does that work?

**Comment by vrv** [comment]
I suspect that unless that email shows up here via Benoit's push, it won't work. :(

Perhaps you could tell Benoit to git commit --amend your commit with the correct email?

**Comment by krikru** [comment]
Now I've sent @benoitsteiner an email.

**Comment by vrv** [comment]
Validated the email addresses manually.

**Pull request merged**